packages/comet-admin-rte/src/core/types.ts

```diff
@@ -1,15 +1,25 @@
 import { SvgIconProps } from "@material-ui/core/SvgIcon";
-import { DraftBlockRenderConfig, DraftInlineStyleType, Editor, EditorState } from "draft-js";
+import { DraftInlineStyleType, Editor, EditorState } from "draft-js";
 
-import { IRteOptions } from "./Rte";
+import { IRteOptions, SupportedThings } from "./Rte";
 
-export interface ICustomBlockType {
-    renderConfig: DraftBlockRenderConfig;
-    label: string;
+// overwrite draftjs' insufficient type for Draft.DraftBlockRenderConfig
+interface DraftBlockRenderConfig {
+    element: string | React.ComponentType;
+    wrapper?: React.ReactNode;
+    aliasedElements?: string[];
 }
 
-export interface ICustomBlockTypeMap {
-    [key: string]: ICustomBlockType;
+export interface IBlocktypeConfig {
+    renderConfig?: DraftBlockRenderConfig; // visual appearance of the blocktype
+    label?: string | React.ReactNode; // displayed in the dropdown
+    group?: "dropdown" | "button"; // displays the element in the dropdown or as button
+    Icon?: (props: SvgIconProps) => JSX.Element;
```

Comment:
```suggestion
    icon?: (props: SvgIconProps) => JSX.Element;
```

Reply:
In this package all prop-keys that are React Components are capitalized:
https://github.com/vivid-planet/comet-admin/blob/ec3ecca7744297d6d5b5ee654e3a9d37f0b3af0c/packages/comet-admin-rte/src/core/types.ts#L21
To be consistent throughout I would leave it like it is in this PR.

But if we do not want this (capitalized prop-keys for react-components), I would refactor this in a seperate PR.

Reply:
The reason why I did this was, because it's nice when you use the icon-prop in JSX in the end. Then the the component is capitalized like all others.
See the use of Item-prop here:
https://github.com/vivid-planet/comet-admin/blob/ec3ecca7744297d6d5b5ee654e3a9d37f0b3af0c/packages/comet-admin-rte/src/core/Controls/ControlButton.tsx#L17

Reply:
alright. makes sense to me!

Reply:
We'll disuss this "convention" briefly: 
https://app-eu.wrike.com/open.htm?id=611182265